cmd/apprepository-controller/cmd/root_test.go

```diff
@@ -134,9 +134,6 @@ func TestParseFlagsCorrect(t *testing.T) {
 			setFlags(cmd)
 			cmd.SetArgs(tt.args)
 			cmd.Execute()
-			serveOpts.ImagePullSecretsRefs = getImagePullSecretsRefs(serveOpts.RepoSyncImagePullSecrets)
-			serveOpts.ParsedCustomAnnotations = parseLabelsAnnotations(serveOpts.CustomAnnotations)
-			serveOpts.ParsedCustomLabels = parseLabelsAnnotations(serveOpts.CustomLabels)
```

Comment:
So it turned out that the test was already ensuring that the options were set correctly, but the test body was explicitly setting them (since the `init` function hadn't done so). Removing these causes the test to fail without the fix.